front/src/app/todo-list/todo-list.component.html

```diff
@@ -8,7 +8,29 @@
     >
       <mat-radio-button [value]="todoList.id" class="task-list__list">{{todoList.title}}</mat-radio-button>
     </a>
+    <button [ngClass]="{'hide-element': !isInputVisible}" mat-icon-button class="task-list__show-input" (click)="showNewTaskGroupFormField()">
+      <mat-icon>add</mat-icon>
+    </button>
+    <form [ngClass]="{'hide-element': isInputVisible}" class="new-task-group">
+      <mat-form-field class="new-task-group__wrapper">
+        <input 
+          [(ngModel)]="inputText"
+          #newTaskGroupFormField type="text" matInput name="newTaskGroupInput" class="new-task-group__input" 
+          (keydown.enter)="createNewTaskGroup(inputText)"
+        >
+        <button [ngClass]="{'hide-element': isInputVisible}" mat-icon-button class="new-task-group__hide-input" (click)="hideNewTaskGroupFormField()">
+          <mat-icon>cancel</mat-icon>
+        </button>

```

Comment:
what I see here doesn't correspond to this button placement in the gif, so what is the final state?